verifier/hash_verifier.go

```diff
@@ -143,7 +143,7 @@ func (v *Verifier) verifyForSingleEvent(event *model.Event) error {
 			logging.Logger.Errorf("error getting challenge result from sp for challengeId: %d, objectId: %s, err=%s", event.ChallengeId, event.ObjectId, err.Error())
 			// TODO: Create error code list for SP side
 			if strings.Contains(err.Error(), "35201") {
-				err := v.dataProvider.UpdateEventStatusVerifyResult(event.ChallengeId, model.Verified, model.BucketDeleted)
+				err := v.dataProvider.UpdateEventStatusVerifyResult(event.ChallengeId, model.BucketDeleted, model.Unknown)
```

Comment:
Why you trust sp's response? If the blockchain says the bucket is existed, then it is existed.

Reply:
On lines 127-133, you should already know whether the bucket exists or not.